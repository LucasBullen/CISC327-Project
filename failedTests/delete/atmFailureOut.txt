3,5c3,4
< Cannot access create feature in atm mode.
< Unrecognized command. Remember all commands must be in lowercase.
< Unrecognized command. Remember all commands must be in lowercase.
---
> Cannot access delete feature in atm mode.
> Logged out.
