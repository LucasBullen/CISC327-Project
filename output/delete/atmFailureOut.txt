Select mode: atm or agent.
Logged into atm mode.
Cannot access create feature in atm mode.
Unrecognized command. Remember all commands must be in lowercase.
Unrecognized command. Remember all commands must be in lowercase.
