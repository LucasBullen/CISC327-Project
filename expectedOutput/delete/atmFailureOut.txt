Select mode: atm or agent.
Logged into atm mode.
Cannot access delete feature in atm mode.
Logged out.
